sqlglot/parser.py

```diff
@@ -1738,39 +1754,54 @@ def parse_option(option, option_lambda):
             )
             parse_option(
                 "not_null",
-                lambda: self._match(TokenType.NOT) and self._match(TokenType.NULL),
+                lambda: self._match(TokenType.NOT)
+                and self._match(TokenType.NULL)
+                and "NOT NULL",
             )
             parse_option(
                 "comment",
                 lambda: self._match(TokenType.SCHEMA_COMMENT) and self._parse_string(),
             )
             parse_option(
                 "primary",
-                lambda: self._match(TokenType.PRIMARY_KEY),
+                lambda: self._match(TokenType.PRIMARY_KEY) and "PRIMARY KEY",
```

Comment:
cheating a little bit with the return value to avoid having to create additional exp classes and handlers in the generator

Reply:
what's going on here?